src/tools/rados/rados.cc

```diff
@@ -142,9 +142,9 @@ void usage(ostream& out)
 "ADVISORY LOCKS\n"
 "   lock list <obj-name>\n"
 "       List all advisory locks on an object\n"
-"   lock get <obj-name> <lock-name>\n"
+"   lock get <obj-name> <lock-name> [--lock-cookie locker-cookie] [--lock-tag locker-tag] [--lock-description locker-desc] [--lock-duration locker-dur] [--lock-type locker-type]\n"
 "       Try to acquire a lock\n"
-"   lock break <obj-name> <lock-name> <locker-name>\n"
+"   lock break <obj-name> <lock-name> <locker-name> --lock-cookie <locker cookie>\n"
```

Comment:
I don't think `lock-cookie` is required, so it should be `[--lock-cookie <locker cookie>]`

Reply:
@dillaman  If locker cookie is empty when acquiring lock, then lock break doesn't need lock-cookie. But if not, lock break needs lock-cookie option to find the correct locker.

Reply:
@david-z that makes sense. but it's still an optional argument. could you update `doc/man/8/rados.rst` to document the connection between these two arguments -- the lock-cookies passed to `lock get` and the one passed to `lock break`?

Reply:
OK, will do.

Reply:
thanks!